packages/graphql/src/services/questions.ts

```diff
@@ -446,3 +450,170 @@ export async function getFileUploadSas(
     fileName: blobName,
   }
 }
+
+export async function updateQuestionInstances(
+  { questionId }: { questionId: number },
+  ctx: ContextWithUser
+) {
+  // fetch the question and return null, if the question does not exist
+  const question = await ctx.prisma.element.findUnique({
+    where: {
+      id: questionId,
+    },
+    include: {
+      instances: {
+        include: {
+          sessionBlock: {
+            include: {
+              session: true,
+            },
+          },
+          stackElement: {
+            include: {
+              stack: {
+                include: {
+                  learningElement: {
+                    where: {
+                      status: DB.LearningElementStatus.DRAFT,
+                    },
+                  },
+                },
+              },
+            },
+          },
+          microSession: {
```

Comment:
add elementStack for the new approach

Reply:
will be added later as this data is not visible in the question pool at the moment and adaptions of these element do not make sense for a user